### ISSUE TITLE
Add arguments to run test or directory like PHPUnit does

### ISSUE THREAD

**Issue by sergeyz** [facile-it/paraunit#81]
Trying to fix some failing tests using `paraunit run`, wondering that arguments are not available for `run` command like PHPUnit. So now I have to change test suite to work with few tests only

```
Usage: phpunit [options] UnitTest [UnitTest.php]
             phpunit [options] <directory>
```

**Comment by Jean85** [comment]
This may need some additional work. Right now, I use PHPUnit class to
select the tests from the test suites. So, will you expect to have this
arguments as an additional filter? In this way it will be ideal, since it
will also work in combination with all the other options.

Il ven 30 dic 2016, 20:45 Sergey Zhuravel <notifications@github.com> ha
scritto:

**Comment by sergeyz** [comment]
Despite `--testsuite` option PHPUnit will run tests from test or folder

**Comment by Jean85** [comment]
Hey, that's done, merged and released with 0.7.4!
